### DISCUSSION TITLE
米家游戏没节奏我不玩

### DISCUSSION THREAD

**Discussion by wshtdgo** [share121/inter-knot#5290] [其他]
最近关于米家各路游戏的节奏越来越多了，之前看着那群sb喷米游是还不忘挂激励我就恶心。喷就喷嘛，还不忘赚钱呢，这不就是无脑蹭还不忘赚钱吗？NPC都要喷，咋滴你是要抽NPC吗？但是最近节奏越来越多，反而我就不难受了，已经变成乐子人了，就专门看网上那群乐子。到现在已经变为，米家游戏没节奏我不玩了（没乐子的游戏不是好游戏）

**Comment by LiuLiBHZ** [comment]
确实

**Comment by Lun603** [comment]
那些噴遊戲的我想遊戲是沒玩的，嘴是沒管好的，腦是不帶的，拜託~一個內容向的遊戲到底有什麼好噴的??

**Comment by NanaChan962** [comment]
这种人我的评价是：我给mhy充钱，我是mhy他爹，黑子靠吃米黑流量赚钱，mhy是黑子他爹，所以我是米黑的爷爷

**Reply by firefly-k423** [reply]
有些人就是这样，你玩米的游戏跟杀了他全家一样，只能说令人费解
